src/test/java/org/wildfly/test/integration/vdx/standalone/InvalidCharactersTestCase.java

```diff
@@ -0,0 +1,156 @@
+package org.wildfly.test.integration.vdx.standalone;
+
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.GroovyXmlTransform;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.Subtree;
+import org.wildfly.extras.creaper.core.offline.OfflineCommand;
+import org.wildfly.test.integration.vdx.TestBase;
+import org.wildfly.test.integration.vdx.category.StandaloneTests;
+import org.wildfly.test.integration.vdx.transformations.DoNothing;
+import org.wildfly.test.integration.vdx.utils.server.Server;
+import org.wildfly.test.integration.vdx.utils.server.ServerConfig;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Path;
+import java.util.Scanner;
+
+/**
+ *
+ * Tests for invalid characters in configuration files
+ *
+ * Created by rsvoboda on 1/18/17.
+ */
+
+@RunAsClient
+@RunWith(Arquillian.class)
+@Category(StandaloneTests.class)
+public class InvalidCharactersTestCase extends TestBase {
+    private static final Path standaloneXml = Server.CONFIGURATION_PATH.resolve("standalone.xml");
+    private static final Path missingStartOfElementStandaloneXml = Server.CONFIGURATION_PATH.resolve("standalone-missingStartOfElement.xml");
+    private static final Path spaceInXmlDeclarationStandaloneXml = Server.CONFIGURATION_PATH.resolve("standalone-spaceInXmlDeclaration.xml");
+    private static final Path missingLastQuoteStandaloneXml = Server.CONFIGURATION_PATH.resolve("standalone-missingLastQuote.xml");
```

Comment:
Name of the file should be "private static final String NAME_OF_XML_FILE = "standalone-....xml" in those 3 paths and used @ServerConfig(configuration = NAME_OF_XML_FILE) annotations below.